Library/Formula/ode.rb

```diff
@@ -23,6 +23,7 @@ class Ode < Formula
   option "with-double-precision", "Compile ODE with double precision"
   option "with-shared", "Compile ODE with shared library support"
   option "with-libccd", "Enable all libccd colliders (except box-cylinder)"
+  option "with-drawstuff", "Include the drawstuff library and demos. Requires X11."
```

Comment:
Aye. `depends_on :x11 if build.with? "drawstuff"` I guess, however much we dislike `if build.with?` dependency statements.